app/views/devise/registrations/_password_details.html.erb

```diff
@@ -1,44 +1,55 @@
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name,  tab: 'password-details-tab'), html: {method: :put, class: "roadmap-form white_background"}) do |f| %>
-  <fieldset class="side-by-side">
-    <legend><%= _('If you would like to change your password please complete the following fields.') %></legend>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name,  tab: 'password-details-tab'), html: {method: :put, id: "password_details_registration_form", class: "form-horizontal"}) do |f| %>
 
-    <%= hidden_field_tag :skip_personal_details, "true" %>
+  <p class="form-control-static"><%= _('If you would like to change your password please complete the following fields.') %></p>
 
-    <div class="form-input">
-      <%= f.label(:current_password, _('Current password'), class: 'required') %>
-      <%= f.password_field(:current_password, class: 'left-indent required input-medium') %>
-      <span role="" id="current_password_error"
-            class="error-tooltip-right left-indent"></span>
+  <%= hidden_field_tag :skip_personal_details, "true" %>
+
+  <div class="form-group">
+    <%= f.label(:current_password, _('Current password'), class: 'col-sm-2 control-label') %>
+    <div class="col-sm-6">
+      <%= f.password_field(:current_password,  class: 'form-control', "aria-required": true ) %>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <%= f.label(:password, _('New password'),  class: 'col-sm-2 control-label') %>
+    <div class="col-sm-6"> 
+      <%= f.password_field(:password,  class: 'form-control', "aria-required": true , id: 'user_new_password') %>
     </div>
+  </div>
 
-    <div class="form-input">
-      <%= f.label(:password, _('New password'), class: 'required') %>
-      <%= f.password_field(:password, class: 'left-indent required input-medium', id: 'user_new_password') %>
-      <span role="" id="password_error"
-            class="error-tooltip-right left-indent"></span>
+  <div class="form-group">
```

Comment:
Copy paste issue here? Don't think you meant to include the current password twice on this form

Reply:
i think this happened by mistake during merge conflicts. fixed.